app/config/routing.yml

```diff
@@ -95,6 +95,21 @@ article-bibtex:
     requirements:
         id: '[a-z0-9-]+'
 
+article-version-pdf:
+    path: /articles/{id}v{version}.pdf
+    defaults:
+        _controller: AppBundle:Articles:pdf
+    requirements:
+        id: '[a-z0-9-]+'
```

Comment:
if an `id` can contain letters, is `42v1` version `1` of article `42` or latest version of article `42v1`?

Reply:
Good question. Pre-existing problem so won't fix here.